.github/workflows/packages-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Packages PR
```

Comment:
Copy paste issue? Should be "Packages CI" correct?

Reply:
Ah yes! Good catch!